backend/dal/Configuration/BaseEntityConfiguration`.cs

```diff
@@ -17,8 +17,8 @@ public virtual void Configure(EntityTypeBuilder<TBase> builder)
             builder.Property(m => m.UpdatedOn).HasColumnType("DATETIME2");
             builder.Property(m => m.RowVersion).IsRowVersion();
 
-            builder.HasOne<User>().WithMany().HasForeignKey(m => m.CreatedById).OnDelete(DeleteBehavior.ClientSetNull);
-            builder.HasOne<User>().WithMany().HasForeignKey(m => m.UpdatedById).OnDelete(DeleteBehavior.ClientSetNull);
+            builder.HasOne(m => m.CreatedBy).WithMany().HasForeignKey(m => m.CreatedById).OnDelete(DeleteBehavior.ClientSetNull);
```

Comment:
What is the motivation for this change?

Reply:
You should be able to update this like this;
```
builder.HasOne(m => m.CreatedBy).WithMany().HasForeignKey(m => m.CreatedById).OnDelete(DeleteBehavior.ClientSetNull);
builder.HasOne(m => m.UpdatedBy).WithMany().HasForeignKey(m => m.UpdatedById).OnDelete(DeleteBehavior.ClientSetNull);
```